### DISCUSSION TITLE
A problem when i use gigahorse

### DISCUSSION THREAD

**Discussion by Ericren2008** [madMAx43v3r/chia-gigahorse#100] [General]
My computer has cpu 2696V2,  256G ram . I use the cmd(cuda_plot_k32 ) -c XX -f XXX -n -1 -C 7 -t D:\plots\ -d D:\farm\ -M 128 -w  . Then the program quit after P1 table3 and there is not some error.    someone help me please!   Thanks !

**Comment by hajes** [comment]
I had something similar. If you farm/plot at same time with C7 on 8GB VRAM...GPU out of memory.  In my case, it was out of memory issue.
